README.md

```diff
@@ -26,6 +26,12 @@ If you prefer not to use a prepackaged binary, or want to have the latest unrele
 * Run `go get github.com/taskcluster/livelog`
 * Run `go get github.com/taskcluster/taskcluster-proxy`
 
+Note that the build process requires custom tooling, and your `go get` may mention errors.
+
+Run `./build.sh` to build binaries, ensure dependencies are vendored, and generate code.
```

Comment:
I would prefer: 

> Run `./build.sh` to check go version, generate code, build binaries, compile (but not run) tests, perform linting, and ensure there are no ineffective assignments in go code.

Note: currently `build.sh` does not check dependencies are vendored, but we could add `dep check` to `build.sh` so that it does, and then keep the "ensure dependencies are vendored" part of your edit, or we could just remove it. _At the moment_ only the CI performs checks dependencies are suitably vendored [here](https://github.com/taskcluster/generic-worker/blob/v14.1.0/.taskcluster.yml#L613), rather than `build.sh` but it would probably be nice to have it in `build.sh` too, as it runs pretty quickly so won't steal a lot of time from the build process.